src/views/static/tools/sanskritToolsUtils.js

```diff
@@ -0,0 +1,53 @@
+import { html } from 'lit-element';
+
+export function     preprocessTaggedString(taggedString){
```

Comment:
Can this not be written much better with a dictionary/object that links things together and running through them like:
```
{
"PPP": "past passive particle (PPP)",
"NC": "common noun",
etc.
}

And then run through all keys to do a replace of that per key and adding a space behind.
```

Reply:
Probably yes. But they would all need to be compiled into global regex, have never done that when iterating a dictionary in JS. In python I would have an idea on how to do it...

Reply:
```
  SanskritTaggerOrder.forEach(result => {
    while (taggedString.includes(result)) {
      taggedString = taggedString.replace(result, SanskritTaggerReplacements[result]);
    }
  })
```
Will push that.